salt/journal/init.sls

```diff
@@ -117,25 +109,39 @@ composer-install:
         {% endif %}
         - cwd: /srv/journal/
         - user: {{ pillar.elife.deploy_user.username }}
-        # to correctly write into var/
-        - umask: 002
         - env:
-            - SYMFONY_ENV: {{ pillar.elife.env }}
             - COMPOSER_DISCARD_CHANGES: 'true'
         - require:
             - file: config-file
-            - journal-cache-clean
```

Comment:
means `composer install` does not need cache cleaning; not sure if we still use hooks or anything similar that relied on that

Reply:
Not any more - https://github.com/elifesciences/journal-formula/pull/60#discussion_r192405407